modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -0,0 +1,42 @@
+process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
+    tag '${meta.id}'
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::atlas-gene-annotation-manipulation=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/atlas-gene-annotation-manipulation%3A1.1.0--hdfd78af_0':
+        'quay.io/biocontainers/atlas-gene-annotation-manipulation:1.1.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(gtf)
+    tuple val(meta), path(fasta)
```

Comment:
You should change it to  
```suggestion
    tuple val(meta2), path(fasta)
```
Otherwise it would clash with the meta information in the gtf channel.